src/main/java/com/bjmbjm/code401d56day12/firstSpringDemo/SentenceReverse.java

```diff
@@ -0,0 +1,44 @@
+package com.bjmbjm.code401d56day12.firstSpringDemo;
+
+public class SentenceReverse {
+    private final long id;
+    private String sentence;
+
+    public SentenceReverse(long id, String sentence){
+        if(id < 0) {
+            throw new IllegalArgumentException("id must be positive a integer.");
+        }
+        this.id = id;
+        if(sentence.equals("") || sentence.equals(" ")){
+            throw new IllegalArgumentException("sentence must contain an alphanumeric character " +
```

Comment:
I realize this makes your string easier to read but it makes your code more inefficient. If you need to, turn on word-wrapping in IntelliJ, it's not the same but it prevents running a loop to combine the two strings